src/views/questionView.js

```diff
@@ -36,7 +45,7 @@ export const createQuestionView = (props) => {
   const showAnswer = (currentQuestion, score) => {
     console.log({ currentQuestion }, score);
     //-y- if the answer is true, scoreDisplay is updated
-    scoreDisplay.innerHTML = score;
+    scoreDisplay.innerHTML = 'Score: ' + score;
 
     const answers = element.querySelectorAll('.answer-item');
     for (let i = 0; i < answers.length; i++) {
```

Comment:
`.querySelectorAll()` returns a NodeList which has a `.forEach()` method that you could use instead of a traditional index based loop. Inside the loop, instead of `anwers[i]` you can then just use `answer`.

Of course, for the embedded loop inside this loop you would need to use a name other than `answer` to avoid confusion.

```suggestion
    answers.forEach((answer) => {
      ...
    }
```